Extensions/Physics3DBehavior/PhysicsCharacter3DRuntimeBehavior.ts

```diff
@@ -0,0 +1,1428 @@
+/// <reference path="./jolt-physics.d.ts" />
+
+namespace Jolt {
+  export interface Body {
+    gdjsAssociatedCharacterBehavior: gdjs.PhysicsCharacter3DRuntimeBehavior | null;
+  }
+}
+
+namespace gdjs {
+  interface PhysicsCharacter3DNetworkSyncDataType {
+    fwa: float;
+    fws: float;
+    sws: float;
+    fs: float;
+    js: float;
+    cj: boolean;
+    lek: boolean;
+    rik: boolean;
+    upk: boolean;
+    dok: boolean;
+    juk: boolean;
+    us: boolean;
+    sa: float;
+    sf: float;
+    tscjs: float;
+    jkhsjs: boolean;
+  }
+
+  export interface PhysicsCharacter3DNetworkSyncData
+    extends BehaviorNetworkSyncData {
+    props: PhysicsCharacter3DNetworkSyncDataType;
+  }
+
+  type Physics3D = {
```

Comment:
A bit of a strange name for a type and an object no?

Reply:
Maybe `Physics3DContext`?